pkg/cmd/connectors/addon/parameters.go

```diff
@@ -0,0 +1,103 @@
+package addon
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/connectors/connectorcmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	connectormgmtclient "github.com/redhat-developer/app-services-sdk-go/connectormgmt/apiv1/client"
+
+	"github.com/spf13/cobra"
+)
+
+// row is the details of a Kafka instance needed to print to a table
+type itemRow struct {
+	ID    string `json:"id" header:"ID"`
+	Value string `json:"value" header:"Value"`
+}
+
+type options struct {
+	outputFormat string
+	id           string
+
+	f *factory.Factory
+}
+
+func NewParametersCommand(f *factory.Factory) *cobra.Command {
+	opts := &options{
+		f: f,
+	}
+
+	cmd := &cobra.Command{
+		Use:     "addon-parameters",
+		Short:   f.Localizer.MustLocalize("connector.addon.params.cmd.shortDescription"),
+		Long:    f.Localizer.MustLocalize("connector.addon.params.cmd.longDescription"),
+		Example: f.Localizer.MustLocalize("connector.addon.params.cmd.example"),
+		Args:    cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, flagutil.ValidOutputFormats...) {
+				return flagutil.InvalidValueError("output", opts.outputFormat, flagutil.ValidOutputFormats...)
+			}
+
+			if opts.id == "" {
+				// TODO validation for id?
```

Comment:
Is there any regexp/pattern that the id should follow?

Reply:
Id is 100% generated I believe. We can check existing ones

Reply:
Operations will return a [404](https://github.com/redhat-developer/app-services-sdk-go/blob/d9a162bd88888915075f3841c508d8be0986da35/.openapi/connector_mgmt.yaml#L682). I will like to avoid an extra request for validation.

Reply:
if not empty is good :D

Reply:
It has been mark required.